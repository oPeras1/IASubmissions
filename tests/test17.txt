52	52	52	52	50	50	50	49	49	43	30	51	51	53	53	54	54	54	54	46
44	52	50	50	50	50	49	49	43	43	30	51	51	53	53	53	46	46	46	46
44	44	50	48	48	48	49	49	43	43	30	51	30	30	30	46	46	47	47	47
39	44	44	48	48	42	42	49	43	40	30	30	30	45	45	46	46	47	47	41
39	39	44	27	27	28	42	42	43	40	40	30	30	30	45	45	46	46	47	41
39	39	27	27	28	28	42	42	43	40	40	36	36	30	38	38	38	38	41	41
39	27	27	27	27	28	28	40	40	40	36	36	36	30	38	38	38	38	37	41
27	27	34	34	27	32	28	28	35	35	36	29	29	30	33	38	37	37	37	31
27	34	34	32	32	32	28	35	35	36	36	36	29	30	33	33	33	37	31	31
27	27	27	24	32	28	28	28	28	28	28	14	29	30	30	33	23	23	31	31
15	27	24	24	20	28	21	21	21	28	28	14	29	19	30	23	23	23	23	31
15	24	24	24	20	25	25	21	21	18	18	14	19	19	22	22	23	26	23	23
15	15	24	20	20	20	25	25	21	18	14	14	19	19	22	23	23	26	26	26
2	15	15	15	15	20	16	11	21	18	18	14	14	19	22	17	23	17	12	12
2	15	15	5	15	15	16	11	11	18	13	13	14	19	6	17	17	17	12	12
2	2	15	5	5	16	16	11	5	5	13	14	14	6	6	6	6	17	12	12
1	2	3	3	5	5	5	11	11	5	13	14	14	6	7	6	6	10	12	12
1	2	2	3	3	5	5	5	11	5	5	9	9	6	7	6	6	10	12	8
1	2	2	3	3	4	5	5	5	5	9	9	6	6	7	6	10	10	10	8
1	2	2	2	3	4	4	4	5	6	6	6	6	6	7	6	8	8	8	8