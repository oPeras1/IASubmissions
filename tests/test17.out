I	I	I	I	T	T	T	L	49	T	30	T	51	L	53	I	I	I	I	46
S	52	50	50	50	T	49	L	T	T	30	T	T	L	L	L	46	46	46	46
S	S	50	48	S	S	L	L	43	T	30	T	30	30	30	46	L	T	T	T
39	S	44	S	S	42	S	49	43	I	L	L	L	S	S	46	L	47	T	41
L	L	44	T	27	L	S	S	43	I	40	30	L	30	S	S	L	L	47	T
L	39	T	T	28	L	42	S	43	I	40	S	S	30	L	38	38	38	T	T
L	27	27	T	27	L	L	40	40	I	S	S	36	30	L	L	L	38	37	T
27	27	S	S	27	S	28	28	S	S	36	L	L	30	S	38	T	T	T	L
27	S	S	32	S	S	28	S	S	36	36	36	L	30	S	S	33	T	31	L
27	27	27	24	S	28	28	28	28	28	28	14	L	30	30	S	23	23	L	L
15	27	S	S	L	28	L	L	L	28	28	14	29	S	30	I	I	I	I	31
15	S	S	24	L	S	S	21	L	I	18	14	S	S	L	L	23	L	23	23
15	15	24	L	L	20	S	S	21	I	14	14	S	19	L	23	23	L	L	L
L	T	T	T	15	20	L	11	21	I	18	14	I	19	L	17	23	T	12	I
L	15	T	5	15	15	L	11	11	I	L	L	I	19	T	17	T	T	12	I
L	L	15	5	5	L	L	S	5	5	L	14	I	T	T	T	6	T	12	I
I	2	L	L	S	S	5	S	S	5	L	14	I	6	I	6	6	L	12	I
I	2	2	L	3	S	S	5	S	5	5	S	S	6	I	6	6	L	12	L
I	2	2	L	3	L	5	5	5	5	S	S	6	6	I	6	L	L	10	L
I	2	2	2	3	L	L	L	5	6	6	6	6	6	I	6	8	8	L	L